src/framework/application.js

```diff
@@ -630,6 +634,22 @@ pc.extend(pc, function () {
             }
         },
 
+        /**
+         * @function
+         * @name pc.Application#getSceneUrl
+         * @description Look up the URL of the scene hierarchy file via the name given to the scene in the editor. Use this to in {@link pc.Application#loadSceneHierarchy}.
+         * @param  {String} name The name of the scene file given in the Editor
+         * @return {String}      The URL of the scene file
+         */
+        getSceneUrl: function (name) {

```

Comment:
Should this live in `app.scenes` instead?

Reply:
I think that would be confusing given that we already have `app.scene` as part of the renderer.